stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/viewholder/MessagePlainTextViewHolder.kt

```diff
@@ -0,0 +1,31 @@
+package io.getstream.chat.android.ui.messages.adapter.viewholder
+
+import android.view.Gravity
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import android.widget.LinearLayout
+import com.getstream.sdk.chat.adapter.MessageListItem
+import io.getstream.chat.android.ui.databinding.StreamItemMessagePlainTextBinding
+import io.getstream.chat.android.ui.messages.adapter.BaseMessageItemViewHolder
+
+public class MessagePlainTextViewHolder(
+    parent: ViewGroup,
+    internal val binding: StreamItemMessagePlainTextBinding =
+        StreamItemMessagePlainTextBinding.inflate(
+            LayoutInflater.from(
+                parent.context
+            ),
+            parent,
+            false
+        )
+) : BaseMessageItemViewHolder<MessageListItem.MessageItem>(binding.root) {
+
+    override fun bindData(data: MessageListItem.MessageItem) {
+        (binding.messageText.layoutParams as LinearLayout.LayoutParams).gravity = if (data.isMine) {
```

Comment:
Would be great to have another property to indicate if the message needs need to be `Gravity.END` or `Fravity.START`
Some customers already asked us to have the possibility to align all messages to the same side. If we add an extra property to our `MessageListItem.MessageItem` that indicate that would be easy to apply changes.
By default this property could be computed using the `MessageListItem.Messageitem.isMin` property

Reply:
We can do it later with style customization support